### PR TITLE
feat: add alpine compatible autosupport binary in containers

### PR DESCRIPTION
**Pull request by rahulguptajss**

### CODE REVIEW

**Comment by rahulguptajss** [comment]
image

**Comment by rahulguptajss** [comment]
version of this asup binary is 21.08.0-7

File: docker/onePollerPerContainer/Dockerfile [hunk 1]
```diff
@@ -21,7 +21,11 @@ RUN make build VERSION=$VERSION RELEASE=$RELEASE
 
 RUN cp -a $BUILD_DIR/harvest.yml $INSTALL_DIR/harvest.yml
 
-RUN cp -aR bin $BUILD_DIR/conf $BUILD_DIR/docs $BUILD_DIR/grafana $BUILD_DIR/autosupport $INSTALL_DIR
+RUN cp -aR bin $BUILD_DIR/conf $BUILD_DIR/docs $BUILD_DIR/grafana $BUILD_DIR/autosupport $INSTALL_DIR &&\
+    rm -f $INSTALL_DIR/autosupport/asup &&\
+    cp $INSTALL_DIR/autosupport/asup_alpine_amd64 $INSTALL_DIR/autosupport/asup &&\
```
Comment (cgrinds) [inline docker/onePollerPerContainer/Dockerfile:26]: Could use mv `$INSTALL_DIR/autosupport/asup_alpine_amd64 $INSTALL_DIR/autosupport/asup` and replace the `cp; rm`

Comment (rahulguptajss) [reply docker/onePollerPerContainer/Dockerfile]: done.

**Review by cgrinds — APPROVED** [review]

**Pull request merged**